pkg/network/ebpf/c/protocols/amqp-helpers.h

```diff
@@ -30,28 +29,47 @@ static __always_inline bool is_amqp(const char* buf, __u32 buf_size) {
        return false;
     }
 
-    uint8_t frame_type = buf[0];
+    __u8 frame_type = buf[0];
```

Comment:
Why are we choosing to prefer the `__ux` type over the `uintx_t` type?

Reply:
sticking with the wide convention in the project
having mixed types only causes an un-readable code